forgerock-authenticator/src/main/java/org/forgerock/android/authenticator/util/MapUtil.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2020 ForgeRock. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package org.forgerock.android.authenticator.util;
+
+import java.util.Map;
+
+/**
+ * Class used to wrap Map functionality.
+ */
+public class MapUtil {
```

Comment:
Make it not public and private constructor for Utility class

Reply:
done